Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG PYTHON_VERSION=3.9
+
+FROM python:${PYTHON_VERSION}-slim
+
+RUN mkdir /covid19-reproductionNumber
+ADD . /covid19-reproductionNumber
+WORKDIR /covid19-reproductionNumber
+
+COPY requirements.txt /covid19-reproductionNumber/requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
The order of these should be inversed to:
1. first add requirements.txt,
2. then run pip install, 
3. and finally add the rest of the files

in such scenario running pip install will be cached unless something changes in requirements.txt (while without inverting the lines it will always run again)